src/Command/GetYmlMetadataCommand.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.IO;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+
+using YamlDotNet;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+
+namespace Microsoft.PowerShell.PlatyPS
+{
+    /// <summary>
+    /// Cmdlet to generate the markdown help for commands, all commands in a module or from a MAML file.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "YamlMetadata", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=2096483")]
```

Comment:
As this file is called `GetYmlMetadataCommand.cs`, it makes sense to have the name for the cmdlet be the same to aid in searches
```suggestion
    [Cmdlet(VerbsCommon.Get, "YmlMetadata", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=2096483")]
```

My other suggestion would be rename this file to `GetYamlMetadataCommand.cs` and add an alias to this command of `Get-YmlMetadataCommand`

Reply:
I've already had a conversation with our documentation folks. Yaml will be the proper term, until they change their mind.